src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -771,4 +790,31 @@ public static String toISO8601(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Returns a JSON array string based on the set.
+     * @param set
+     * @return json array string
+     */
+    public static String toJson(final Set<String> set) {
+        if(set == null || set.isEmpty()) {
+            return "";
+        }
+
+        // We want to avoid using Json libraries cause
+        // may cause incompatibilities on different Jenkins versions.
+        final StringBuilder sb = new StringBuilder();
```

Comment:
can there be quotes in Jenkins labels?

Reply:
If you mean double quotes: no, Jenkins removes the double quotes (`"`) when you configure the labels for the worker.
Other characters are allowed (`'`, `$`, `%`, etc..)